app/views/case_groups/_form.html.erb

```diff
@@ -26,7 +26,7 @@
       <%= form.label :casa_case_ids, 'Cases' %>
       <%= form.select(
         :casa_case_ids,
-        current_organization.casa_cases.pluck(:case_number, :id),
+        current_organization.casa_cases.map { |casa_case| [sanitize("#{casa_case.case_number} - #{render AssignedVolunteersComponent.new(casa_case, current_user) }"), casa_case.id] },
```

Comment:
Here a method on CasaCase makes even more sense. Rendering html that will then get turned into a string anyway, overcomplicates the code and puts undue load on the server.

Reply:
"Rendering html that will then get turned into a string anyway, overcomplicates the code and puts undue load on the server." what does this mean?